glusterd2/transaction/transaction.go

```diff
@@ -26,9 +26,11 @@ type Txn struct {
 	reqID uuid.UUID
 	Ctx   TxnCtx
 	Steps []*Step
-	// Nodes should be a union of the all the TxnStep.Nodes and must be set
-	// before calling Txn.Do(). This is currently only used to determine
-	// liveness of the nodes before running the transaction steps.
+
+	CheckAlive bool
```

Comment:
Instead of `CheckAlive` it would be better to have a `DontCheckAlive`. This way the nil value is always false, and you would only need to set the value when you don't want the check to happen.

It is always nicer to keep the defaults to something most operations would use, and only have those operations that need it different, change it to what they want.